modules/removing-bare-metal-hosts-from-provisioner.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/managing-bare-metal-hosts.adoc
+
+:_content-type: PROCEDURE
+[id="removing-bare-metal-hosts-from-provisioner_{context}"]
+= Removing bare metal hosts from the provisioner node
+
+In certain circumstances it is desirable to temporarily remove bare metal hosts from the provisioner node.
+For example, during provisioning when a bare metal host reboot is triggered by using the {product-title} administration console or as a result of a Machine Config Pool update, {product-title} logs into the integrated Dell Remote Access Controller (iDrac) and issues a delete of the job queue.
+
+To prevent management of the number of `Machine` objects that matches the number of available `BareMetalHost` objects, add a `baremetalhost.metal3.io/detached` annotation to the `MachineSet` object.
+[NOTE]
+====
+This annotation only has an effect for `BareMetalHost` objects that are in either `Provisioned`, `ExternallyProvisioned` or `Ready/Available` state.
```

Comment:
s/only has an effect for/has an effect for only (keep _only_ next to the word it modifies)